src/Autocomplete.js

```diff
@@ -23,37 +24,35 @@ class Autocomplete extends Component {
 
   renderDropdown (data) {
     const { value } = this.state;
+    const { minLength } = this.props;
 
-    let li = null;
-    if (value.length) {
-      li = Object.keys(data).map((key, idx) => {
-        const index = key.toUpperCase().indexOf(value.toUpperCase());
-        if (index !== -1 && value.length < key.length) {
-          return (
-            <li key={key + '_' + idx}>
-              {data[key] ? <img src={data[key]} className="right circle"/> : null}
-              <span>
-                {index !== 0 ? key.substring(0, index) : ''}
-                <span className='highlight'>{value}</span>
-                {key.length !== index + value.length ? key.substring(index + value.length) : ''}
-              </span>
-            </li>
-          );
-        }
-      })
+    if (minLength && minLength > value.length) {
+      return null
```

Comment:
I'm doing the eslint and building the testing suite for this now, so this is already fixed in the next update.